state-synchronizer/src/request_manager.rs

```diff
@@ -252,10 +253,10 @@ impl RequestManager {
         }
 
         let req_info = self.add_request(req.known_version, peers.clone());
-        debug!(
-            "[state sync] request next chunk - chunk req info: {:?}",
-            req_info
-        );
+        debug!(log
+            .clone()
```

Comment:
Does this actually need to be cloned?

Reply:
yeah, it potentially gets moved later in the code